### ISSUE TITLE
Quest Bug Wynn Excavation Site A

### ISSUE THREAD

**Issue by Kroko-fant** [Wynncraft/Issues#7142]
Hello Wynncraft Team
My Questbook didn't update....
for video: https://youtu.be/HdiZLF-9Y3s
When I talk to Vade he tells me to enter the tomb by throwing an item behind a wall
and when I do that the hole opens BUT There came the Slogan A strong force prevents you from entering.
So because my Questbook didn't upgrade I can't go on

**Comment by DebugingLonks** [comment]
The book does not ever say 'quest book updated' in this quest, I meant to add that as an issue earlier. Easy way to fix (and a glitch lol) if you are a mage you can teleport in even without starting quest by running then looking down + teleporting. If you are a diff class I have no suggestions.

**Comment by Kroko-fant** [comment]
I'm Warrior

**Comment by DebugingLonks** [comment]
Yeah, no solution for you... also its not a slogan. Wynn's slogan is "Soon™"

**Comment by DebugingLonks** [comment]
#6866

**Comment by Moe-Ronickah** [comment]
@Kroko-fant , the barrier effect fails to get removed sometimes, here.  You should try switching servers and trying again.  Usually, this issue fixes itself when a world gets reloaded.
